core-bundle/src/Resources/public/core.js

```diff
@@ -581,20 +706,20 @@ var AjaxRequest =
 			div.getParent('div').getElement('pre').toggleClass('disabled');
 		}
 
-		icon = image.get('data-icon') || AjaxRequest.themePath + 'icons/visible.svg';
-		icond = image.get('data-icon-disabled') || AjaxRequest.themePath + 'icons/invisible.svg';
```

Comment:
Why did you change the existing `toggleVisibility` method? Shouldn't it remain untouched for BC?

Reply:
Fixed in 818a5f18c87cf97383fdac47406feecb1626e553